appinventor/components/tests/com/google/appinventor/components/runtime/AnomalyDetectionTest.java

```diff
@@ -0,0 +1,65 @@
+package com.google.appinventor.components.runtime;
+
+import com.google.appinventor.components.runtime.util.YailList;
+import gnu.lists.Pair;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class AnomalyDetectionTest extends RobolectricTestBase {
+  AnomalyDetection anomalyDetection = new AnomalyDetection(getForm());
+  YailList xList = YailList.makeList(Arrays.asList("1", "2", "3","4","5","6"));
+  YailList yList = YailList.makeList(Arrays.asList("1", "2", "3","2","2","88"));
```

Comment:
You should put the initialization into a `setUp()` function and make sure to call `super.setUp()`.